### PR TITLE
Bnb/outputs

### PR DESCRIPTION
**Pull request by bnb32**
Moving reV base output handler to rex. We're using it in the sup3r work and it makes more sense for this to be in rex given its general use case. Also would be nice to avoid a reV dependency.

### CODE REVIEW

**Review by grantbuster — CHANGES_REQUESTED** [review]
I think this makes sense, just make sure you update the rex version so we can increment the requirements files. Also, we can just subclass the outputs handler in the downstream repo's to maintain the same import path and to update the handler version attrs. 

We're still going to need the reV dependency for the pipeline utilities which should probably not be moved to rex.

**Comment by ppinchuk** [comment]
> Also, we can just subclass the outputs handler in the downstream repo's to maintain the same import path and to update the handler version attrs.

Makes sense... I assume we will want to do this in `reV` then? Should be fairly simple but we will probably have to wait until this PR is merged and the new rex version is released.

**Comment by bnb32** [comment]
Yeah that's what I was thinking as well. Just import Outputs from rex and leave all the subclasses in reV alone.

**Review by grantbuster — APPROVED** [review]

**Pull request merged**
